src/components/Signup/SignupForm.js

```diff
@@ -46,13 +46,15 @@ class SignupForm extends Component {
     return (
       <form
         onSubmit={handleSubmit(submitSignup)}
-        className="form"
+        className="form sign-up-form"
       >
-        <Field name="email" component={FormField} type="email" label="Email" />
-        <Field name="username" component={FormField} type="text" label="Username"/>
-        <Field name="password" component={FormField} type="password" label="Password" />
-        <Field name="password_confirmation" component={FormField} type="password" label="Password Confirmation" />
-        <button type="submit">Submit</button>
+        <Field name="name" component={FormField} type="text" label="NAME"/>
+        <Field name="email" component={FormField} type="email" label="EMAIL"/>
+        <Field name="password" component={FormField} type="password" label="PASSWORD"/>
+        <Field name="password_confirmation" component={FormField} type="password" label="CONFIRMATION PASSWORD"
+               placeholder="Min. 6 characters long"/>
```

Comment:
When the lenght of a JSX line exceeds 80 lines, I prefer separating props by indented newlines, like this:

```
<Field
  name="password_confirmation"
  component={FormField}
  type="password"
  label="CONFIRMATION PASSWORD"
  placeholder="Min. 6 characters long"
/>
```
instead of this
```
<Field name="password_confirmation" component={FormField} type="password" label="CONFIRMATION PASSWORD"
  placeholder="Min. 6 characters long"/>
```

However, that's a preference of mine. You can choose the other way, but remember to be consistent with it.